src/microbe_stage/Microbe.Contact.cs

```diff
@@ -269,6 +275,14 @@ public void ApplyMembraneWigglyness()
             Membrane.Type.MovementWigglyness) * 0.2f;
     }
 
+    /// <summary>
+    ///   Give this microbe a specified amount of invulnerability time. Overrides previous value.
+    /// </summary>
```

Comment:
Could you add a remark here that the invulnerability doesn't apply to all damage sources?

Reply:
Yeah I think I mentioned it in a different comment (?) but I should have specified it more clearly

Reply:
Well my concern here is that someone will read this documentation on this method and think they can use it in some different context, but when they try to use it, then it doesn't work at all for non-pilus damage. That's why I think the documentation on this method should be clearer.

Reply:
Yeah, I agree. I added a small clarification in the latest commit.

Reply:
This overtly specific comment line (281) still exists...

Reply:
What do you mean? I made the comment more specific to say that only Pili have implemented i-frames. The line 281 should specify this now.

Reply:
I mean that this method documentation should mention that this doesn't apply to all damage sources without specifically saying the pilus is the one this applies to. Because in the future:
- Someone will update the invulnerability to also apply to some other thing
- That person will forget to update the comment here, and if I don't catch it in a code review, the outdated comment will be still here
- Someone else will be doing work on Thrive and read the documentation on this method, and read that only pilus is affected
- That someone may waste multiple hours of their time and sanity due to an outdated comment

That's why I like to avoid unnecessarily specific comments that can easily get outdated.

Reply:
The comment has been changed to refer to pili (for some reason GitHub is showing an outdated version on the issue discussion but not on the diff page). Do you still want it to be changed again?

Reply:
Well the version shown on the diff page is the one I take issue with. The one shown here on the discussion page actually looks better. For absolutely clarify I checked out the code locally and this is the not that good version I see:
![kuva](https://user-images.githubusercontent.com/3796411/205162297-bae4b687-5725-48fb-94e3-1e4c0bf07807.png)

Reply:
Looks like @Enovale has updated it now.